Rules/Rules.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <VersionPrefix>1.18.3</VersionPrefix>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <VersionPrefix>1.19.0</VersionPrefix>
+    <TargetFrameworks>netcoreapp3.1;netstandard2.0;net452</TargetFrameworks>
     <AssemblyName>Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules</AssemblyName>
     <PackageId>Rules</PackageId>
     <RootNamespace>Microsoft.Windows.PowerShell.ScriptAnalyzer</RootNamespace> <!-- Namespace needs to match Assembly name for ressource binding -->
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Engine\Engine.csproj" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-    <PackageReference Include="Microsoft.Management.Infrastructure" Version="1.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Microsoft.Management.Infrastructure" Version="2.0.0" />
```

Comment:
As far as I see, PowerShell 6.2.4 still references version 1.0.0 of this package, therefore would this not be a breaking change?
https://github.com/PowerShell/PowerShell/blob/v6.2.4/src/System.Management.Automation/System.Management.Automation.csproj#L25

Reply:
will fix